.github/workflows/scripts/show_logs.sh

```diff
@@ -18,7 +18,7 @@ kubectl get events --sort-by='.metadata.creationTimestamp'
 echo ::endgroup::
 
 echo ::group::OBJECTS
-kubectl get ${TEST:-pulp},pvc,configmap,serviceaccount,secret,networkpolicy,ingress,service,deployment,statefulset,hpa,job,cronjob -o yaml
+kubectl get ${TEST:-pulp},pvc,configmap,serviceaccount,secret,networkpolicy,ingress,service,deployment,statefulset,hpa,job,cronjob -o yaml -l owner!=helm
```

Comment:
Helm uses [secret as default storage driver](https://helm.sh/docs/faq/changes_since_helm2/#secrets-as-the-default-storage-driver) to keep the release information. Because of that, GH workflow is getting stuck when we try to get the output from it (the secret data stored in sh.helm.release.v1 is very large).